api/v1beta1/hyperconverged_types.go

```diff
@@ -507,6 +507,14 @@ type HyperConvergedFeatureGates struct {
 	// +kubebuilder:default=false
 	// +default=false
 	EnableApplicationAwareQuota *bool `json:"enableApplicationAwareQuota,omitempty"`
+
+	// primaryUserDefinedNetworkBinding deploys the needed configurations for kubevirt users to
+	// be able to bind their VM to a UDN network on the VM's primary interface.
+	// Note: this feature is in Developer Preview.
+	// +optional
+	// +kubebuilder:default=false
+	// +default=false
```

Comment:
Please update the default values also for the `featureGates` field in the `HyperConvergedSpec` struct, and the `spec` field in the `HyperConverged` struct.

Reply:
Not sure what you mean there.. Can you elaborate?

Reply:
the default values appears also in the places I mentioned above.

Reply:
DONE